mlir/include/mlir/Dialect/Tosa/Transforms/Passes.h

```diff
@@ -33,7 +33,8 @@ void populateTosaDecomposeDepthwise(MLIRContext *ctx,
 void populateTosaFoldConstantPatterns(MLIRContext *ctx,
                                       RewritePatternSet &patterns,
                                       bool foldSplatOrSingleUseOnly,
-                                      bool enableIntCastFolding);
+                                      bool enableIntCastFolding,
+                                      bool enableTileFolding);
```

Comment:
Sorry, can you bite the bullet and change this to take `TosaLayerwiseConstantFoldPassOptions` instead of four booleans? Otherwise its too easy to mess the call site up, especially when merging.